app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :created_games, :class_name => 'Game' 
```

Comment:
we'd need to tell rails here how to find the user from the game.. 

lets add sth like: 

```
has_many :created_games, :class_name => 'Game', foreign_key: "owner_id" 
has_many :joined_games, :class_name => 'Game',  foreign_key: "opponent_id"
```